ceno_zkvm/examples/riscv_opcodes.rs

```diff
@@ -277,9 +284,12 @@ fn main() {
         let timer = Instant::now();
 
         let transcript = Transcript::new(b"riscv");
+
+        let span = entered_span!("CREATE_PROOF");
```

Comment:
Perhaps something more descriptive?  (Just like you did for the outer spans?)

```suggestion
        let create_proof = entered_span!("CREATE_PROOF");
```

Reply:
Renamed in most cases :+1